espanol/canudos.md

```diff
@@ -9,3 +9,5 @@ el Consejero:
 [Llegó el periodista miope con noticias importantes.](noticias/noticias.md)
 
 [Se dirigió a la muralla exterior del pueblo.](muralla/muralla.md)
+
+[Mando a buscar al cocinero.](cocinero/cocinero.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.